### DISCUSSION TITLE
Differences between Frigate's implementation of go2rtc and standalone go2rtc as Home Assistant Add-on?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#10349] [General]
Is there a difference between how Frigate configures a camera for use in its built-in go2rtc instance versus how it is configured in a standalone instance of go2rtc? The reason for this question is that I am able to get a camera to live stream in Frigate through its go2rtc, but I **am not** able to get the same camera two work using the same camera configuration yaml code in the go2rtc add-on in Home Assistant. I've posted the issue over at the github for go2rtc (https://github.com/AlexxIT/go2rtc/issues/992) but figured I ask here too since it involves Frigate.

**Comment by NickM-27** [comment]
the code of go2rtc itself is identical. Frigate does however apply some default configs https://github.com/blakeblackshear/frigate/blob/dev/docker/main/rootfs/usr/local/go2rtc/create_config.py

**Reply by Cold-Lemonade** [reply]
Please pardon my ignorance if this question has an obvious answer. 

Is it possible to view the go2rtc config file that this py script produces? If so, where do I look within my docker container for it?

**Reply by NickM-27** [reply]
You can see it inside of the go2rtc dashboard config page

**Reply by Cold-Lemonade** [reply]
Sorry, but I don't know how to access the "go2rtc dashboard config page" through Frigate. What's the url for it?

**Reply by NickM-27** [reply]
You'll need to map and access port 1984

**Reply by Cold-Lemonade** [reply]
Thank you! That was exactly what I needed.

Unfortunately, if I use the same config file in my go2rtc add-on in Home Assistant, it does not work. I wonder why?

**Reply by Cold-Lemonade** [reply]
I thought I got it working, but I was wrong. Still not working. What a bummer.
